src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/advice/GlobalExceptionHandler.java

```diff
@@ -68,6 +69,13 @@
         return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
     }
 
+    @ExceptionHandler(CaseAlreadyExistsException.class)
+    ResponseEntity<Object> handleCaseAlreadyExistsException(CaseAlreadyExistsException exception) {
```

Comment:
GlobalExceptionHandler now has a handler for BaseException, see comment for CaseAlreadyExistsException